tilavarauspalvelu/api/graphql/types/reservation/types.py

```diff
@@ -239,3 +284,47 @@ def resolve_calendar_url(root: Reservation, info: GQLInfo) -> str:
         calendar_url = reverse("reservation_calendar", kwargs={"pk": root.pk})
         signature = ical_hmac_signature(f"reservation-{root.pk}")
         return f"{scheme}://{host}{calendar_url}?hash={signature}"
+
+    def resolve_pindora_info(root: Reservation, info: GQLInfo) -> PindoraInfoData | None:
+        # No Pindora info if access type is not 'ACCESS_CODE'
+        if root.access_type != AccessType.ACCESS_CODE:
```

Comment:
Let's add: or the reservation is in the past.

Reply:
Could there be some edge case, since access code can be valid some time after reservation, that the reserve would need to enter the premises, e.g. to fetch something that they forgot, and couldn't if they couldn't see the access code any more?

Reply:
There could. I suppose allowing the Pindora info to be fetched for +24 hours since `reservations.ends` would cover those situations (while avoiding making unnecessary calls when past reservations are looked at).

Reply:
Added